src/Temporalio/Worker/ActivityWorker.cs

```diff
@@ -183,6 +183,7 @@ private void StartActivity(Bridge.Api.ActivityTask.ActivityTask tsk)
                 HeartbeatDetails: start.HeartbeatDetails,
                 HeartbeatTimeout: OptionalTimeSpan(start.HeartbeatTimeout),
                 IsLocal: start.IsLocal,
+                Priority: new(start.Priority),
```

Comment:
I don't really think it's a big deal, but a GC/instance-count conscious .NET dev (there are a lot of them) would have something like `start.Priority is { } p ? new(p) : Priority.Default`, meaning a static readonly `Default` on the `Priority` class that is just `new()` (especially since the object is immutable).